tools/ci_build/github/azure-pipelines/nuget/cpu-mklml-pipeline.yml

```diff
@@ -10,5 +9,5 @@ variables:
 jobs: 
 - template: templates/cpu-mklml.yml
   parameters:
-    AgentPool : $(AgentPoolWin)
+    AgentPool : 'Win-CPU-2019'
```

Comment:
Does this agentpool exist on AIInfra vso instance (for ESRP signing)?

Reply:
YES